faststream/broker/core/usecase.py

```diff
@@ -342,3 +342,8 @@ async def publish(
             publish = partial(m(None).publish_scope, publish)
 
         return await publish(msg, **kwargs)
+    
+    @abstractmethod
+    async def ping(self, timeout: int | None) -> bool:
```

Comment:
`int | None` is not compatible with lt python3.10. Use `Optional[int]` instead (btw, should it be `float` instead of `int`?)